Models/JsonStat/SSBAgesInNorway.cs

```diff
@@ -0,0 +1,31 @@
+using System.Text.Json.Serialization;
+
+namespace Backend.Models.JsonStat
+{
+    // Remove comments when merge-ready
+    // Based on https://data.ssb.no/api/v0/dataset/1074.json?lang=no
+    public class SSBAgesInNorway
+    {
+        /*
+         * I tried abstracting this class as much as possible, so that the only thing
+         * that has to be changed when creating a new JSON-stat object, only the dimension
+         * needs to be changed.
+         */
```

Comment:
I believe you can use `///` to create MSDocs (I think that's what they are called) to provide other developers with extra information regarding a class/property/function/&c.

Reply:
Indeed, but that's mainly used for comments that should be in the code, even after merging.
E.g.
```cs
/// <summary>
/// Squares the input integer
/// </summary>
/// <param name="x">Input integer</param>
/// <returns>The square of the input</returns>
public int Square(int x)
{
    return x * x;
}
```

Reply:
Fair point! Thought these were supposed to be in `master`. _After rebase_